wsf_scraping/settings.py

```diff
@@ -80,10 +80,13 @@
 FEED_TEMPDIR = 'var/tmp/'
 
 if FEED_CONFIG == 'AWS':
-    AWS_S3_BUCKET = 'data-labs'
-    AWS_S3_FILE_NAME = 'scraper-results/%(name)s - %(time)s.json'
+    AWS_S3_BUCKET = 'data-labs-data'
```

Comment:
why did we change the bucket? do we use the `data-labs` bucket as well?

Reply:
This change is to be more consistent with the other buckets created for the datalabs aws account.